app/src/main/res/layout/activity_view_location.xml

```diff
@@ -6,40 +6,48 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <include
+        android:id="@+id/include"
+        layout="@layout/app_header" />
+
     <fragment
+        android:id="@+id/google_map"
+        android:name="com.google.android.gms.maps.SupportMapFragment"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:id="@+id/google_map"
-        android:name="com.google.android.gms.maps.SupportMapFragment" />
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
         android:id="@+id/view_address_text"
         android:layout_width="match_parent"
         android:layout_height="60dp"
         android:layout_marginHorizontal="8dp"
-        android:layout_marginTop="8dp"
-        android:hint="Autofilled"
+        android:layout_marginTop="12dp"
+        android:background="@drawable/drop_shadow"
         android:gravity="center_vertical"
-        android:paddingLeft="12dp"
-        android:background="@color/colorBackground"
+        android:hint="Autofilled"
         android:imeOptions="actionDone"
         android:inputType="text"
+        android:paddingLeft="12dp"
         android:textSize="20dp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toBottomOf="@+id/include" />
 
-    <Button
-        android:id="@+id/scan_isbn_button"
-        android:layout_width="match_parent"
-        android:layout_height="60dp"
-        android:layout_marginHorizontal="36dp"
-        android:layout_marginBottom="16dp"
-        android:backgroundTint="@color/colorPrimary"
-        android:text="@string/scan_isbn"
-        android:textAllCaps="false"
-        android:textColor="@color/colorBackground"
-        android:textSize="24dp"
-        app:layout_constraintBottom_toBottomOf="parent" />
+    <!--    <Button-->
```

Comment:
Is there a reason for keeping this? Are we going to potentially need this in the future?

Reply:
I wasn't sure if we wanted this feature as it would require the view location activity to handle scan tasks.

Reply:
I see. It wouldn't be a bad idea to leave it in but maybe add a TODO that reflects that potential feature?

Reply:
Thanks will do!